# origin coordinates of the object
float64 origin_x
float64 origin_y
float64 origin_z

# dimensions of the object
float64 angle_difference

# destination coordinates of the object
float64 destination_x
float64 destination_y
float64 destination_z

---
bool finished